IMAPlugin.swift

```diff
@@ -414,7 +419,7 @@ enum IMAState: Int, StateProtocol {
         }
         
         if let bitrate = self.config?.videoBitrate {
-            self.renderingSettings.bitrate = bitrate
+            self.renderingSettings.bitrate = Int(bitrate)
```

Comment:
why the cast to Int?

Reply:
Compilation error:
Cannot assign value of type 'Int32' to type 'Int'